### DISCUSSION TITLE
setTimeout

### DISCUSSION THREAD

**Discussion by dudatom4** [JetClient/jet-client-support#69] [General]
Hi there. I just want to ask you if there is a way how to setTimeout after/before any request. 
I want to create a funkcion that waits for data in response. 
But its seems like this command doesnt exists here.
Example
`setTimeout(() => {
  console.log("Hello World!");
}, 500);`
Thanks for reply

**Comment by AntonShuvaev** [comment]
Hi! Currently, the `setTimeout` function is not available. However, I'm planning a significant update in the next couple of weeks which will include support for `setTimeout` among other enhancements.

**Comment by dudatom4** [comment]
Thanks for info. I´ll be waiting for it. It would be nice :)

**Comment by AntonShuvaev** [comment]
Version 2024.2.0 now includes support for `setTimeout` and `clearTimeout`.
